### PR TITLE
Try to deploy to both sites

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -89,9 +89,12 @@ lazy val herokuDeployAndRelease =
...
       "heroku container:login",
-      "docker tag noirlab/gpp-obs registry.heroku.com/${{ secrets.HEROKU_APP_NAME }}/web",
-      "docker push registry.heroku.com/${{ secrets.HEROKU_APP_NAME }}/web",
-      "heroku container:release web -a ${{ secrets.HEROKU_APP_NAME }} -v"
+      "docker tag noirlab/gpp-obs registry.heroku.com/${{ secrets.HEROKU_APP_NAME_GN }}/web",
+      "docker push registry.heroku.com/${{ secrets.HEROKU_APP_NAME_GN }}/web",
+      "heroku container:release web -a ${{ secrets.HEROKU_APP_NAME_GN }} -v",
+      "docker tag noirlab/gpp-obs registry.heroku.com/${{ secrets.HEROKU_APP_NAME_GS }}/web",
+      "docker push registry.heroku.com/${{ secrets.HEROKU_APP_NAME_GS }}/web",
+      "heroku container:release web -a ${{ secrets.HEROKU_APP_NAME_GS }} -v"
```
[the remark this answers is not in the document]
Comment (cquiroz) [reply build.sbt:97]: I haven't tried anything yet, should we try this version first?

[the remark this answers is not in the document]
Comment (rpiaggio) [reply build.sbt:97]: `docker push` seems to have a `-a` modifier that pushes all tags

[the remark this answers is not in the document]
Comment (rpiaggio) [reply build.sbt:97]: > I haven't tried anything yet, should we try this version first?

ok

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**